Bookcrossing.Data/Repositories/PublisherRepository.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Bookcrossing.Contracts.Abstractions.RequestFeatures;
+using Bookcrossing.Contracts.Enumeration;
+using Bookcrossing.Data.DbContext;
+using Bookcrossing.Data.Entities;
+using Bookcrossing.Data.Repositories.Interface;
+using Microsoft.EntityFrameworkCore;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Bookcrossing.Data.Repositories
+{
+    public class PublisherRepository : RepositoryBase<Publisher>, IPublisherRepository
+    {
+        public PublisherRepository(BookcrossingDbContext dbContext)
+            : base(dbContext)
+        { }
+
+        public async Task<IReadOnlyCollection<Publisher>> GetAsync(AuthorPublisherParams parameters, CancellationToken ct = default)
+        {
+            parameters.SearchString ??= string.Empty;
```

Comment:
probably uesless?

Reply:
when you add white space it comes as null
sttring.contains(null) == exception